CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Genesis Sample Theme Changelog
 
+## [3.1.0] - Unreleased
```

Comment:
@dreamwhisper Should we set the expected release date here now? (We could adjust if it changes.)

```suggestion
## [3.1.0] - 2019-08-21
```